vocabularies/Hierarchy.xml

```diff
@@ -265,6 +265,42 @@ The template actions themselves cannot be invoked.</String>
         </Property>
       </ComplexType>
 
+      <Function Name="EvaluateHierarchy" IsBound="true" EntitySetPath="Node">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Evaluates a node in a recursive hierarchy and adds the [`RecursiveHierarchy`](#RecursiveHierarchy) annotation" />
+        <Annotation Term="Core.LongDescription">
+          <String>Can be used to determine, for example, the `LimitedRank` of a newly created or copied node
+in the JSON batch request:
+```json
+{"requests": [{
+  "id": "1",
+  "method": "post",
+  "url": "HierarchyDirectory(1)/Nodes('A')/CopyAction"
+}, {
+  "id": "2",
+  "dependsOn": ["1"],
+  "method": "get",
+  "url": "$1/Hierarchy.EvaluateHierarchy(
```

Comment:
Don't we need more information for a LimitedRank? What about the Levels parameter? And can we really replace the InputSet by a single node?

Reply:
One entity is perfectly fine, but do we have all information at hand to compute e.g. the LimitedRank? Wouldn't that depend on Levels as well? And doesn't it also depend on the input set (you could filter it beforehand, at least in future)?